Google.Api.Generator/ProtoUtils/ProtoExtensions.cs

```diff
@@ -53,6 +53,16 @@ public static string CSharpNamespace(this FileDescriptor desc)
 
         public static string CSharpFieldName(this FieldDescriptor field) => field.Name.ToLowerCamelCase();
 
+        public static string CSharpName(this EnumValueDescriptor desc)
```

Comment:
Sorry for the duplicate code here :(
It might be worth linking to the C++ for reference. For example, link to https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/compiler/csharp/csharp_helpers.cc and suggest finding GetEnumValueName.

(I suspect this code may need to be adjusted to be a closer copy of that, too.)

Reply:
Thanks for the reference. That prefix removal is a little more complex than I expected! I'll update the code here.